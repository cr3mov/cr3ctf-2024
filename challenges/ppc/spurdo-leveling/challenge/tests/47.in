199456418773268789 942797896045260525
2 1072
4 1971
