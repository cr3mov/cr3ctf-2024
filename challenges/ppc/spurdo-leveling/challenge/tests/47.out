244198908433640 244198908433638 244198908433637
